### ISSUE TITLE
memory or cpu question

### ISSUE THREAD

**Issue by 544531645** [viromedia/virocore#167]
when I not open virocore , I do audio convert is fast , when I open virocore then do audio format is slow , for I use ffmpeg demo do wav->m4a (11s audio need near 5s), in arproject do wav->m4a (11s audio need near 15s)， virocore use so much thear and memory ?

**Comment by 544531645** [comment]
@dthian

**Comment by VikAdvani** [comment]
Hi @544531645, memory use depends on how many models, textures and other assets you load for your AR Scene. You can use the android profiler in android studio to see the memory usage. If your scene is using high resolution textures plus complicated models w/ animations then memory use will likely be high. 

CPU usage is also driven by how complicated your scene is. If you have a scene with animations, lighting and shadows then cpu use will be higher vs a more simple scene.

**Comment by 544531645** [comment]
yes ,I have animations, lighting and shadows , but when I convert audio so slow , It affects the user experience, but again on ios, scenekit's speed is not affected at all. Is there a good solution?

**Comment by VikAdvani** [comment]
Hi @544531645, the speed and performance depends on the phone you are using. What phone are you seeing this type of performance on? On older android phones, I'd recommend leaving out some features(such as shadows and complicated lighting, use flat lighting). On newer Android devices(like Samsung s8 and higher) then you can include more complicated scenes. Is the audio conversation slow on some devices or all of them?
